functions.sh

```diff
@@ -169,6 +169,18 @@ function get_versions() {
   fi
 }
 
+function is_alpine() {
+  local variant
+  variant=${1}
+  shift
+
+  if [ ! "${variant}" = "${variant#alpine}" ]; then
+    return 0
+  fi
+
+  return 1
```

Comment:
In this case, I this we can reverse the condition above, and simply return 1 in it?